aten/src/ATen/native/native_functions.yaml

```diff
@@ -6010,15 +6010,10 @@
 - func: scatter_add.dimname(Tensor self, Dimname dim, Tensor index, Tensor src) -> Tensor
   variants: function, method
 
-- func: _scatter_reduce(Tensor self, int dim, Tensor index, str reduce, *, int? output_size=None) -> Tensor
-  structured_delegate: _scatter_reduce.out
+- func: scatter_reduce.two(Tensor self, int dim, Tensor index, str reduce, *, int? output_size=None) -> Tensor
```

Comment:
Hey @mikaylagawarecki! It looks like this PR promotes `scatter_reduce` to a public API (you can now call `torch.scatter_reduce` from python), but it doesn't have any public documentation.

For the release, we want to either avoid adding it to python, or adding docs for it. Do you have a preference? If you want to hide it from python, I can make a PR. But if you want to add docs, you'll need to cherry-pick the docs into the release branch (I can point you to the process internally).

cc @albanD

Reply:
Yep that should be documented and cherry picked in the release branch. (or we can revert this for the release if the doc is not ready)

cc @cpuhrsch

Reply:
Let's make this private again. We didn't land on a conclusion around this function in time. Thanks for checking @bdhirsh @albanD .